erpnext/accounts/report/general_ledger/general_ledger.js

```diff
@@ -146,6 +146,12 @@ frappe.query_reports["General Ledger"] = {
 				return frappe.db.get_link_options('Project', txt);
 			}
 		},
+		{
+			"fieldname": "include_dims",
+			"label": __("Include Accounting Dimensions in Ledger"),
```

Comment:
```suggestion
			"label": __("Show Accounting Dimensions"),
```

Reply:
I am changing to "Consider Accounting Dimensions," as it's not simply a matter of showing the dimensions or not; the actually general ledger lines produced by the report change depending on whether or not this box is checked.